internal-server.md

```diff
@@ -131,8 +131,7 @@ To download an existing repository use `git clone`:
 ```
 $ git clone git@data.neuro.polymtl.ca:datasets/data-single-subject
```

Comment:
currently testing-- getting this:
~~~
julien-macbook:~/code/spine-generic $ git clone git@data.neuro.polymtl.ca:datasets/data-single-subject
Cloning into 'data-single-subject'...
FATAL: autocreate denied

fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
~~~

i'm under VPN

Reply:
We renamed that one to "data-single-subject_DO-NOT-USE.git".

Reply:
ok, in that case i suggest choosing another example data for the doc, eg: `git@data.neuro.polymtl.ca:datasets/basel-mp2rage`

Reply:
I'm going to change this to an obviously fake dataset instead.

Reply:
Or actually I went with `sct-testing-large`, since that one seems like it'll stick around for a long time.